src/index.tsx

```diff
@@ -189,13 +196,25 @@ export default class InlineSVG extends React.PureComponent<Props, State> {
   private request = () => {
     const { cacheRequests, fetchOptions, src } = this.props;
 
+    if (typeof fetch !== 'function') {
+      return this.handleError(new Error('fetch is not a function'));
+    }
+
     try {
       if (cacheRequests) {
         cacheStore[src] = { content: '', status: STATUS.LOADING };
       }
 
-      return fetch(src, fetchOptions)
-        .then(response => {
+      const fetchWithRetry = require('fetch-retry')(fetch, {
+        retryOn: (attempt: number, error: any, response: Response) => {
+          // Retry on any network error, or 4xx or 5xx status codes,
+          // as long as we don't exceed the maximum retries.
+          return (error !== null || response.status >= 400) && attempt <= MAX_RETRIES;
```

Comment:
return explicit value (true/false), I think that if the condition is clear the comment is not necessary because it doesn’t add any content, only noise